libs/core/src/lib/utils/directives/resize/resize.directive.ts

```diff
@@ -71,6 +76,11 @@ export class ResizeDirective implements OnChanges, AfterContentInit, OnDestroy {
 
     /** @hidden */
     ngAfterContentInit(): void {
+        if (this._rtlService) {
+            this._isRtl = this._rtlService.rtl.getValue();
```

Comment:
Not sure you need to get this value, because rtl property is a BehaviorSubject

Reply:
Let it be like

```
get _isRtl(): boolean {
  return this._rtlService?.rtl.getValue()
}
```